src/Symfony/Component/Notifier/Bridge/Matrix/Exception/UnsupportesMsgTypeByAPIException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Matrix\Exception;
+
+use Symfony\Component\Notifier\Bridge\Matrix\MatrixTransport;
+use Symfony\Component\Notifier\Exception\LogicException;
+
+class UnsupportesMsgTypeByAPIException extends LogicException
+{
+    public function __construct(string $given)
+    {
+        parent::__construct(\sprintf('Unsupported message type: "%s". Only %s are supported by Matrix Synapse Client-Server API v3', $given, implode(', ', MatrixTransport::SUPPORTED_MSG_TYPES_BY_API)));
+    }
```

Comment:
Not sure about this dedicated exception, why if the version changes (v3) or if more types are supported in the future than in our constant?